src/frontend/screens/PhotoPreviewModal/index.tsx

```diff
@@ -0,0 +1,497 @@
+
+        return photo.type === 'processed' ? (
+          <TouchableOpacity
+            onPress={() => {
+              navigation.navigate('ConfirmDeletePhoto', {
+                photo,
+                onSuccess: () => {
+                  navigation.goBack();
+                },
+              });
+            }}
+            style={{
+              borderStyle: 'solid',
+              borderWidth: 1,
+              borderColor: WHITE,
+              borderRadius: 24,
+              flexDirection: 'row',
+              justifyContent: 'center',
+              alignItems: 'center',
+              paddingHorizontal: 8,
+              paddingVertical: 6,
+            }}>
+            <MaterialIcons name="delete" size={18} color={WHITE} />
+            <Text style={{marginLeft: 4, color: WHITE, fontSize: 13}}>
+              {intl(m.headerDeleteButtonText)}
+            </Text>
+          </TouchableOpacity>
+        ) : null;
+      },
+    };
+  };
+}
+
+function CreatedByDeviceIdInfoItem({
+  projectId,
+  observationDocId,
+}: {
+  projectId: string;
+  observationDocId: string;
+}) {
+  const lang = useAppLanguageTag();
+
+  const {data: observation} = useSingleDocByDocId({
+    projectId,
+    docType: 'observation',
+    docId: observationDocId,
+    lang,
+  });
+
+  const {data: createdByDeviceId} = useDocumentCreatedBy({
+    projectId,
+    originalVersionId: observation.originalVersionId,
+  });
+
+  return (
+    <InfoItem
+      icon={
+        <MaterialIcons
+          name="devices"
+          size={20}
+          color={NEW_DARK_GREY}
+          allowFontScaling
+        />
+      }>
+      <BodyText selectable style={styles.primaryInfoText} numberOfLines={1}>
+        {
+          // TODO: Showing this ID is a temporary measure
+          // Ideally we use a format that is more human readable/friendly
+          createdByDeviceId.slice(0, 15)
+        }
+      </BodyText>
+    </InfoItem>
+  );
+}
+
+function getDeviceDetailsText({make, model}: {make?: string; model?: string}) {
+  const displayedParts: Array<string> = [];
+
+  if (make) {
+    displayedParts.push(make);
+  }
+
+  if (model) {
+    displayedParts.push(model);
+  }
+
+  if (displayedParts.length === 0) {
+    return null;
+  }
+
+  return displayedParts.join(' ');
+}
+
+function getCameraDetailsText(
+  {
+    fNumber,
+    orientation,
+  }: {
+    fNumber?: number;
+    orientation?: number;
+  },
+  formatMessage: IntlShape['formatMessage'],
+): string | null {
+  const displayedParts: Array<string> = [];
+
+  if (typeof orientation === 'number') {
+    try {
+      const layout = getPhotoLayout(orientation);
+
+      displayedParts.push(
+        layout === 'horizontal'
+          ? formatMessage(m.landscape)
+          : formatMessage(m.portrait),
+      );
+    } catch (err) {
+      captureException(err);
+
+      // TODO: What happens if layout cannot be determined or is missing?
+      // Should we still try to display something?
```

Comment:
highlighting this. unlikely we'll run into this but basically if the orientation value from the EXIF is unrecognized, we won't attempt to show an orientation to the user (e.g. no `Portrat` or `Landscape` text will show).